src/components/AddCustomerForm.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import "./AddCustomerForm.css";
+
+const AddCustomerForm = props => {
+  const initUser = {
+    id: null,
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: " ",
+    checkInDate: "",
+    checkOutDate: ""
+  };
+
+  const [customer, setCustomer] = useState(initUser);
+
+  const handleChange = event => {
+    const updateCustomer = {
+      ...customer,
+      [event.target.name]: event.target.value
+    };
+    setCustomer(updateCustomer);
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    console.log(customer.email);
+    if (
+      customer.email &&
+      customer.firstName &&
+      customer.surname &&
+      customer.title &&
+      customer.roomId &&
+      customer.checkInDate &&
+      customer.checkOutDate
+    ) {
+      handleChange(event, props.addCustomer(customer));
```

Comment:
Not sure why you're invoking `handleChange` inside of the `handleSubmit` function. Also `handleChange` is designed to take 1 argument and you're passing 2 arguments.

Reply:
Yes probably no need to pass an event here.Thank you.